src/System.IO.Compression/src/System.IO.Compression.builds

```diff
@@ -5,6 +5,9 @@
     <Project Include="System.IO.Compression.csproj">
       <AdditionalProperties>TargetGroup=</AdditionalProperties>
     </Project>
+    <Project Include="System.IO.Compression.csproj" Condition="'$(TargetsWindows)' == 'true'">
```

Comment:
I'm working on a change to change how we condition this so my change should take care of all these cases.